src/constructs/ecs/ecs-task.ts

```diff
@@ -200,6 +202,7 @@ export class GuEcsTask extends Construct {
 
     const task = new EcsRunTask(scope, `${id}-task`, {
       cluster,
+      subnets: { subnets },
```

Comment:
I think we can be more opinionated here, and always place the task in the private subnet (similar to how we implement EC2 apps). WDYT?

Reply:
Were you imagining something like this?
 
```typescript
const privateSubnets = GuVpc.subnetsFromParameter(scope, { type: SubnetType.PRIVATE, app });

const task = new EcsRunTask(scope, `${id}-task`, {
    cluster,
    subnets: { subnets: privateSubnets },
    ...
}
```
This would effectively make the `subnets` prop redundant, since we're defining the subnets in the pattern constructor.

Reply:
All done, thanks. Have also added a couple of tests.
Let me know if I need to do anything else before I merge! :-)